enterprise/views.py

```diff
@@ -486,6 +489,15 @@ def get_enterprise_course_enrollment_page(self, request, enterprise_uuid, course
             logger.error('Unable to find enterprise customer for UUID: %s', enterprise_uuid)
             raise Http404
 
+        if EnterpriseCourseEnrollment.objects.filter(
+                enterprise_customer_user__enterprise_customer=enterprise_customer,
+                enterprise_customer_user__user_id=request.user.id,
+                course_id=course_id
+        ).exists():
```

Comment:
@douglashall That's a good question - I'm not sure if this case was decided on, so I made a comment on the upstream ticket asking about what the behavior should be: https://openedx.atlassian.net/browse/ENT-355?focusedCommentId=263107&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-263107